numba_cuda/numba/cuda/cudadrv/enums.py

```diff
@@ -309,7 +309,10 @@
 # Applicable options: PTX compiler options, ::CU_JIT_FALLBACK_STRATEGY
 CU_JIT_INPUT_LIBRARY = 4
 
-CU_JIT_NUM_INPUT_TYPES = 6
+# LTO IR
+CU_JIT_INPUT_LTO_IR = 5
+
+CU_JIT_NUM_INPUT_TYPES = 7
```

Comment:
This should not be bumped - there are still 6 input types (it's just that 5 (LTO-IR) was never used before so it didn't appear in Numba)